proteus/Domain.py

```diff
@@ -301,6 +302,45 @@ def writeAsymptote(self, fileprefix):
     def writeXdmf(self,ar):
         raise UserWarning("Xdmf output not implemented")
 
+def unitSimplex(nd=2):
+    """Builds a 2 or 3 dimension reference element.
+    
+    Parameters
+    ----------
+    nd : int
+        The elements dimension (must be 2 or 3)
+
+    Returns
+    -------
+    reference_element
+    """
+    if nd!=2 and nd!=3:
+        logEvent("ERROR - Reference element must have dimension 2 or 3")
+        sys.exit(1)
```

Comment:
You could also raise an exception here. Maybe a ValueError (https://docs.python.org/2/library/exceptions.html). Nathan was  thinking about this some.